ktor-server/ktor-server-netty/jvm/test/io/ktor/tests/server/netty/NettyHttp2LocalConnectionPointTest.kt

```diff
@@ -0,0 +1,152 @@
+/*
+ * Copyright 2014-2020 JetBrains s.r.o and contributors. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package io.ktor.tests.server.netty
+
+import io.ktor.http.*
+import io.ktor.server.netty.http2.*
+import io.netty.handler.codec.http2.*
+import java.net.*
+import kotlin.test.*
+
+class NettyHttp2LocalConnectionPointTest {
+    @Test
+    fun testMethod() {
+        val point = point {
+            method("PUT")
+        }
+
+        assertEquals(HttpMethod.Put, point.method)
```

Comment:
This and some of the following tests don't relate to changed code

Reply:
@rsinukov since we can't agree with @Stexxe  about extra tests I've added, could you please state your thoughts: is it acceptable to commit all the tests suite or only the relevant test? The reason why I did this is that the overhead for splitting this, creating branch, ticket and PR just for few related tests.

Reply:
I can be wrong, but I think we agreed on covering with unit tests parts of the project that you touched. IMO creating extra ticket just for tests is redundant, because they can't break any functionality.